packages/backend/src/clients/Slack/SlackClient.ts

```diff
@@ -54,10 +54,16 @@ export class SlackClient {
                 organizationUuid,
             );
 
-        return new WebClient(installation?.token);
+        if (!installation) {
+            throw new Error('Could not find slack installation');
+        }
```

Comment:
the above model call `getInstallationFromOrganizationUuid` was throwing before, so im preserving the behavior here.